Libraries/Text/Text/RCTTextView.m

```diff
@@ -341,6 +359,26 @@ - (NSString *)accessibilityLabel
 }
 #endif // TODO(macOS GH#774)
 
+# pragma mark - Selection
+
+#if TARGET_OS_OSX // TODO(macOS GH#774)
+- (void)textDidEndEditing:(NSNotification *)notification
+{
+  _textView.selectedRange = NSMakeRange(NSNotFound, 0);
+}
+
+- (void)drawFocusRingMask
+{
+  if ([self enableFocusRing]) {
+    NSRectFill([self bounds]);
+  }
+}
```

Comment:
In our fork we removed the focus ring for selectable text since text was always un-focusable & never a key view (cc. @lyahdav). 

I believe the the behavior we want is when the TextView is `selectable`:
  1. it becomes a key view
  2. Is selectable with keyboard 
  3. Has a focus ring when focused

NOTE: I've added an extra text input to this example to verify key focus behavior.
![CleanShot 2022-09-15 at 16 43 55](https://user-images.githubusercontent.com/96719/190527383-e1845937-650c-4796-b6f4-ec17f26b90c7.gif)

Reply:
@Saadnajmi thoughts on this? The default NSTextView behavior for selectable non-editable text is that it is in the key view loop, but tabbing to it doesn't show a focus ring, which is not ideal. In our fork we subclass NSTextView and make canBecomeKeyView return NO always. It's still accessible. IMO it doesn't make sense to have it be a tab stop. This would also put it in line with NSTextField's behavior when it's selectable and non-editable. Alternatively one option would be to expose a focusable property on Text for controlling this.

Reply:
We have at least one client team that _wants_ Text to be a tab stop so they can focus on it, then do Cmd+C to copy text. So I think we should at least support `focusable`. IIRC, `selectable` effectively made it focusable in the old behavior. The "new" behavior having the two props handle separate things makes sense. 
@Ppatboyd FYI, since he worked on similar stuff on the windows (well, win32) side

Reply:
I'm pretty sure even if it is a tab stop, when you're focused on an NSTextView and press Cmd+C it won't copy the text because it's not selected yet. I think you need to first press Cmd+A to select all, then Cmd+C. @shwanton perhaps you can verify?

Reply:
Correct, you will still need to select the text before it can be copied. It would be possible to automatically select the content when tabbed, but that is not expected behavior.

I'l add support for the `focusable` prop that will allow tab stops on TextViews.